src/core/terms.ml

```diff
@@ -399,7 +399,7 @@ let _Wild : tbox = Bindlib.box Wild
 let _TRef : term option ref -> tbox = fun r ->
   Bindlib.box (TRef(r))
 
-(** [_LLet t a u] lifts let binding [let x := t : a in u<x>]. *)
+(** [_LLet t a u] lifts let binding [let x := t : a in u<x>]. - @PROBLEM ? *)
```

Comment:
`@problem`

Reply:
It's a bit confusing with the comment in the type `term`

Reply:
`(** [_LLet t a u] lifts a let node to the {!type:tbox} type, given a
    boxed term [t] for the definition, a boxed term [a] for the type, and a boxed binder [u]. *)`

Reply:
Btw, there is a comment missing for _Impl: `(** [Impl a b] lifts a non-dependent product node to the {!type:tbox} type, given
    a boxed term [a] for the domain of the product, and a boxed term [b] for
    its codomain. *)`

Reply:
Done